Cargo.toml

```diff
@@ -108,7 +108,7 @@ ron = "0.6.2"
 serde = { version = "1", features = ["derive"] }
 # Needed to poll Task examples
 futures-lite = "1.11.3"
-crevice = {path = "crates/crevice"}
+crevice = { path = "crates/crevice", version = "0.8.0", features = ["glam"] }
```

Comment:
Specifying a version here will make cargo use the non-vendored version when publishing.

Reply:
There are 3 dependencies on crevice in Bevy crates, it was the only one not specifying a version so I added it for consistency.

Before Bevy release, Cart will also have to rename it to "bevy_crevice" or something

Reply:
That depends whether we even need to vendor it anymore. I think we should at least try it from crates.io after the recent rewrite and see if it fits our needs or not.

Reply:
I tried, it's missing the array PR. once that's merged and release we wouldn't need to vendor it

Reply:
Perfect! :)